docs/design-archive/koord-runtime-proxy-design-doc.md

```diff
@@ -129,7 +129,7 @@ TODO
 Under containerd scenario, to make koord-runtime-proxy a proxy between kubelet and containerd, kubelet parameters should be altered as shown
 below:
 ```
-kubelet <other options> --container-runtime=remote --container-runtime=/var/run/koord-runtimeproxy/runtimeproxy.sock
+kubelet <other options> --container-runtime=remote --container-runtime-endpoint=/var/run/koord-runtimeproxy/runtimeproxy.sock
```

Comment:
hey,  which version introduced the container-runtime-endpoint parameter?

Reply:
> hey, which version introduced the container-runtime-endpoint parameter?

this flag was introduced all the wayyyyyy back to 2017

Reply:
OK… my mistake